### DISCUSSION TITLE
Anyone deployed on AWS with lambda ? AVG costs

### DISCUSSION THREAD

**Discussion by pullmann4rent** [remix-run/remix#8490] [Q&A]
Hello,

How much would be the cost to deploy an remix app with aws lambda that can reach 5k r/s ? I know there is no exactly price cost but what you think would it cost?

**Comment by flexdinesh** [comment]
It's going to depend on what type of website and routes you have. Other factors might also influence your cloud bill depending on your deployment setup - CloudFront (if you need a distributed cache), S3 (if you want to serve your assets from a static storage which is usually cheaper), etc.

Only you can predict your cloud bill. Here are some tips to calculate them:

- Will all your requests be rendered request time (SSR and no cloudfront caching)? Then all your cost will be [lambda pricing](https://aws.amazon.com/lambda/pricing/). This will include assets as well. Each asset load will invoke your lambda.
- Will you upload your assets to S3 and only serve the web pages from Lambda? Then your assets cost will be based on [S3 pricing](https://aws.amazon.com/s3/pricing/) and web page cost will be based on [lambda pricing](https://aws.amazon.com/lambda/pricing/)
- Will you have a CloudFront distribution in front of your lambda and cache 80% of your requests (this is what you should be doing anyway)? Then 80% of your cost will be [CloudFront pricing](https://aws.amazon.com/cloudfront/pricing/) and 20% will be [lambda pricing](https://aws.amazon.com/lambda/pricing/).

**Reply by smcstewart** [reply]
And you can also use the [AWS Cost Calculator](https://calculator.aws) to put all your costs together into a single estimate. But, it's on you to define how your app will use whatever services. There are loads of considerations for you, but am sure you can give it a quick go in the calculator for a start.
